README.md

```diff
@@ -25,7 +25,7 @@ The easiest way to get started with development is to use Visual Studio Code wit
 - [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
 - Docker
   - For Linux, macOS, or Windows 10 Pro/Enterprise/Education use the [current release version of Docker](https://docs.docker.com/install/)
-  - Windows 10 Home requires [WSL 2](https://docs.microsoft.com/windows/wsl/wsl2-install) and the current Edge version of Docker Desktop (see instructions [here](https://docs.docker.com/docker-for-windows/wsl-tech-preview/)). This can also be used for Windows Pro/Enterprise/Education.
+  - Windows 10 Home requires [WSL 2](https://docs.microsoft.com/windows/wsl/wsl2-install) and [Docker Desktop WSL 2 backend on Windows](https://docs.docker.com/desktop/features/wsl/). This can also be used for Windows Pro/Enterprise/Education.
```

Comment:
- The old link redirected to the link I'm suggesting. It seems that this feature is no longer limited to Edge.
- Links should be accessible to the blind and thus not use `here` as their titles.